server/config/toml.go

```diff
@@ -86,6 +86,10 @@ index-events = {{ .BaseConfig.IndexEvents }}
 # It works when tendermint's prometheus option (config.toml) is set to true.
 prometheus = {{ .BaseConfig.Prometheus }}
 
+# ChanCheckTxSize is he size of RequestCheckTxAsync of BaseApp
```

Comment:
1. typo: ChanCheckTxSize is "the" ~
2. For readability, I think it's necessary to synchronize the comment of `config.go` and `toml.go` for `chan-check-tx-size`.

Reply:
Thank you for checking the typo. And the comment of `config.go` is generated from Ostracon. So it is out of the scope of this PR. But I'll add the comment to need to check the mempool size of the `config.go`.